airflow-core/tests/unit/plugins/test_plugin.py

```diff
@@ -102,12 +102,14 @@ async def dispatch(self, request, call_next):
     "name": "Name of the Middleware",
 }
 
-iframe_view_with_metadata = {
+external_view_with_metadata = {
     "name": "Test IFrame Plugin",
-    "src": "https://www.google.com",
+    "href": "https://www.google.com",
     "icon": "https://example.com/icon.svg",
-    "url_route": "/test_iframe_plugin",
+    "dark_mode_icon": "https://example.com/dark_icon.svg",
+    "url_route": "test_iframe_plugin",
     "destination": "nav",
+    "category": "browse",
 }
```

Comment:
I realized that Google blocks (probably because of security?) to embed the homepage into an iFrame. Therefore I tested (positive) with these parameters:
```
external_view_with_metadata = {
    "name": "Test IFrame Airflow Docs",
    "href": "https://airflow.apache.org/",
    "icon": "https://raw.githubusercontent.com/lucide-icons/lucide/refs/heads/main/icons/plug.svg",
    "url_route": "test_iframe_plugin",
    "destination": "nav",
    "category": "browse",
}
```
Did not find a comparable dark mode icon but maybe it would be best to move-away from google as example.

Reply:
You're right, that's a good point, thanks Jens, I think all of those reference to google were in 'tests' but that dosn't cost much to remove the ambiguity and use Airflow Docs framing. I'll update the example to reference dirrectly that.